gnosis/safe/api/transaction_service_api/transaction_service_api.py

```diff
@@ -416,3 +416,21 @@ def post_message_signature(
                 f"Error posting message signature: {response.content}"
             )
         return True
+
+    def decode_data(
+        self, data: Union[bytes, HexStr], to_address: Optional[ChecksumAddress] = None
+    ) -> Dict[str, Any]:
+        """
+        Retrieve decoded information using tx service internal ABI information given the tx data.
+
+        :param data: tx data as a 0x prefixed hexadecimal string.
+        :param to_address: address of the receiving contract
+        :return:
```

Comment:
What are you returning? Maybe you can define a TypedDict (inside a file `entities.py` on the `transaction_service_api/` folder) to make clear what's the return value. And we should do the same for other functions

Reply:
You are right, I did this by looking at the other methods, but it is not the right thing to do. I will try to update the other method signatures.